capsules/src/l3gd20.rs

```diff
@@ -377,9 +377,9 @@ impl Driver for L3gd20Spi<'_> {
     ) -> ReturnCode {
         match subscribe_num {
             0 /* set the one shot callback */ => {
```

Comment:
Just FYI this is a known rustfmt issue: https://github.com/rust-lang/rustfmt/issues/4119